game/src/info/trip.rs

```diff
@@ -386,6 +389,329 @@ pub fn cancelled(ctx: &mut EventCtx, app: &App, id: TripID) -> Widget {
     Widget::col(col)
 }
 
+/// Highlights intersections which were "slow" on the map
+fn highlight_slow_intersections(app: &App, details: &mut Details, id: TripID, ctx: &EventCtx) {
+    let data_opt = &app.primary.sim.get_analytics().trip_intersection_delays(id);
+    if data_opt.is_some() {
+        let data = data_opt.unwrap();
+        for (id, time) in data {
+            let intersection = app.primary.map.get_i(id.parent);
+            // Maybe alter the delay times
+            let (normal_delay_time, slow_delay_time) = if intersection.is_traffic_signal() {
+                (30, 120)
+            } else {
+                (5, 30)
+            };
+            let (fg_color, bg_color) = if time < &normal_delay_time {
+                (Color::WHITE, app.cs.normal_slow_intersection)
+            } else if time < &slow_delay_time {
+                (Color::BLACK, app.cs.slow_intersection)
+            } else {
+                (Color::WHITE, app.cs.very_slow_intersection)
+            };
+            // Draw label after, to prevent being obscured by the line
```

Comment:
I'm confused by this comment, what's the "line" in this context?

Reply:
It's the lane highlight on the map.
Like the dotted line/solid line.
What do you think would make it clearer?
Or is it even necessary?

Reply:
I think you can remove the comment or mention something near where `highlight_slow_intersections` is called after `make_timeline`. That's where the ordering matters.

Reply:
Actually, do you think it would be worth moving `highlight_slow_intersections` and `highlight_slow_lanes` to the bottom of `make_timeline`?

Reply:
That'd be more clear, yeah. (Assuming you mean moving the calls and not inlining the function's contents)

Reply:
Yeah moving the calls. -Done